week-2/errors/0.js

```diff
@@ -4,6 +4,10 @@
 // interpret the error message and figure out why it's happening, if your prediction was wrong
 
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+   str = `${str[0].toUpperCase()}${str.slice(1)}`;
```

Comment:
Excellent job! You can also return the result string directly like:
`return `${str[0].toUpperCase()}${str.slice(1)}`;`